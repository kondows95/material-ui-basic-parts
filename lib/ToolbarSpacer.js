"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
var react_1 = __importDefault(require("react"));
var styles_1 = require("@material-ui/core/styles");
var useStyles = styles_1.makeStyles(function (theme) { return ({
    toolbar: theme.mixins.toolbar,
}); });
var ToolbarSpacer = function () {
    var classes = useStyles();
    return react_1.default.createElement("div", { className: classes.toolbar });
};
exports.default = ToolbarSpacer;
