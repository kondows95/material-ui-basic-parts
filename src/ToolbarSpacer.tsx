import React from 'react';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
    toolbar: theme.mixins.toolbar,
}));

const ToolbarSpacer = (): React.ReactElement => {
    const classes = useStyles();
    return <div className={classes.toolbar} />;
};

export default ToolbarSpacer;
