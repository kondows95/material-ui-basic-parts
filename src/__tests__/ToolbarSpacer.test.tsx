import React from 'react';
import TestLib from 'react-testing-library-wrapper';
import ToolbarSpacer from '../lib/ToolbarSpacer';

describe('Basic Usage', () => {
    let tLib: TestLib;
    beforeEach(() => {
        tLib = new TestLib(<ToolbarSpacer />);
    });

    it('Snapshot', () => {
        expect(tLib.render.asFragment()).toMatchSnapshot();
    });
});
