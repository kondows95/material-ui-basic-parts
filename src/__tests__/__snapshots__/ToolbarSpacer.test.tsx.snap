// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic Usage Snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-toolbar-1"
  />
</DocumentFragment>
`;
